### ISSUE TITLE
Text fails WCAG accessibility

### ISSUE THREAD

**Issue by matgargano** [huja2502/fun-with-git#1]
Royal Blue (#4169e1) with a foreground of black fails using https://webaim.org/resources/contrastchecker/
